commands-yml/commands/mobile-command.yml

```diff
@@ -84,6 +84,7 @@
       | mobile:acceptAlert | Accepts an on-screen alert | Optional button label to click on | <none> |
       | mobile:dismissAlert | Dismisses an on-screen alert | Optional button label to click on | <none> |
       | mobile:scrollBackTo | Scroll from one element to another | `{elementId, elementToId}` | `{elementId: 2, elementToId: 1}` |
+      | mobile:scroll | Scroll the given scrollable element until the element identifier by `strategy` and `selector` becomes visible since Appium 1.18.0. `maxSwipes` can limit the number of scroll action. Please read `mobileScroll` section in [appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver/blob/master/lib/commands/general.js) for more details. | `{strategy, selector, maxSwipes}` | `{strategy: 'accessibility id', selector: 'target element', maxSwipes: 1}` |
```

Comment:
.. the number of scroll actions...

Reply:
please remove maxSwipes from the default example. It is fine to autodetect it.

Reply:
the `scroll` endpoint can also accept an optional `element` property, which is the element identifier, which is expected to be scrolled. Without this property set the first scrollable container on the page will be selected by default